src/ChannelServer/Util/GmCommands.Handlers.cs

```diff
@@ -48,6 +48,7 @@ public GmCommands()
 			Add("go", "<destination>", this.HandleGo);
 			Add("goto", "<team name>", this.HandleGoTo);
 			Add("recall", "<team name>", this.HandleRecall);
+			Add("recallmap", "", this.HandleRecallMap);
```

Comment:
It might be useful to have take an optional parameter, so you can recall everybody on a specific map to you. For example, to get everyone out of an event map.

Reply:
Good idea, I'll implement that.